config/locales/en.yml

```diff
@@ -25,8 +25,27 @@ en:
       strip_insignificant_zeros: false
       unit: "$"
 
-  titles:
-    application: Budgetapp
+  home:
+    show:
+      link: "New Account"
+      new_account: "To sign up for a new account click the 'New Account' button"
+      welcome: "Welcome to Budget App"
+
+  users:
+    defaults:
```

Comment:
Looking at the Rails guides, it suggests to have the following structure:
```
en:
  activerecord:
    models:
      user: Dude
    attributes:
      first_name: First name
```

Reply:
I am currently following Rails Guides 4.1.4 "Lazy" Lookup:
```Rails implements a convenient way to look up the locale inside views. When you have the following dictionary:
en:
  books:
    create:
      success: Book created!
you can look up the books.index.title value inside app/views/books/index.html.erb template like this (note the dot):
<%= t '.title' %>```

Reply:
👍